### ISSUE TITLE
.Net 3.1 image security update about [DSA 5111-1] zlib security update 

### ISSUE THREAD

**Issue by LianwMS** [dotnet/core#7372]
### Description

We are using mcr.microsoft.com/dotnet/aspnet:3.1 image as our base image. Recently, we always get the report from S360 that the image need security update of https://lists.debian.org/debian-security-announce/2022/msg00079.html. 

We want to confirm that whether the latest image contains the update. if not, will it be contained later?

**Comment by joperezr** [comment]
@MichaelSimons @mthalman could you please take a look at this one?

**Comment by mthalman** [comment]
Yes, the latest version of the image contains the fixed version:

```console
$ docker pull mcr.microsoft.com/dotnet/aspnet:3.1
3.1: Pulling from dotnet/aspnet
Digest: sha256:1687e296970d9123b1d67d79ca01b871024f66d851defc50d207346e01c68096
Status: Image is up to date for mcr.microsoft.com/dotnet/aspnet:3.1
mcr.microsoft.com/dotnet/aspnet:3.1

$ docker run --rm mcr.microsoft.com/dotnet/aspnet:3.1 apt list zlib1g

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Listing...
zlib1g/now 1:1.2.11.dfsg-1+deb10u1 amd64 [installed,local]
```

According to https://security-tracker.debian.org/tracker/CVE-2018-25032, `1:1.2.11.dfsg-1+deb10u1` is the fixed version:
![image](https://user-images.githubusercontent.com/15789599/163459829-fa437dff-c320-4c02-ae84-0d5cf52abd89.png)

**Comment by joperezr** [comment]
thanks @mthalman, closing this for now as the original question was answered. @LianwMS feel free to reopen if you have any follow up questions.

**Comment by Tyler-H** [comment]
Late to the party, but zlib didn't fix the vuln until version 1.2.12: https://github.com/madler/zlib/blob/master/ChangeLog#L59

Is zlib1g a fork of the zlib project or something?
